sherpa/models/model.py

```diff
@@ -540,6 +540,12 @@ def __setstate__(self, state):
     def __getitem__(self, filter):
         return FilterModel(self, filter)
 
+    def check_regrid_kwargs(self, **kwargs):
```

Comment:
I can understand pushing this into the `ArithmeticModel` class, but the name is a bit odd as we don't have the concept of regrid at this level of the API.

Reply:
One could introduce a class called ```RegriddableModel```, from which the classes ```RegriddableModel1D``` and ```RegriddableModel2D``` inherit from and the method ```check_regrid_kwargs``` live there?